examples/siro_sandbox/sandbox_app.py

```diff
@@ -728,6 +903,69 @@ def evaluate_cubic_bezier(ctrl_pts, t):
     return result
 
 
+def _ease_fn_in_out_quat(t: float):
```

Comment:
Move this math helper outside this file. Let's avoid bloating `SandboxDriver` further.